### DISCUSSION TITLE
Upgrade of minor releases - 2.0.3. to 2.1.1.

### DISCUSSION THREAD

**Discussion by robs-fotoalbum** [typemill/typemill#438] [Q&A]
Hi, how do I perform an upgrade of minor releases? Do I have to do a fresh installation each time or is it sufficient to just copy the new systems folder to the existing 2.x.x. version via ftp?

**Comment by trendschau** [comment]
Hi Rob, yes, just the system folder, See the Details here: https://typemill.net/getting-started/update
